deployer/src/deployment/run.rs

```diff
@@ -379,23 +394,52 @@ async fn load(
                 info!("successfully loaded service");
             }
 
-            let resources = response
+            let resources: Vec<_> = response
                 .resources
                 .into_iter()
                 .map(|res| {
                     let resource: resource::Response = serde_json::from_slice(&res).unwrap();
-                    record_request::Resource {
-                        r#type: resource.r#type.to_string(),
-                        config: resource.config.to_string().into_bytes(),
-                        data: resource.data.to_string().into_bytes(),
-                    }
+                    resource
+                })
+                .collect();
+
+            let loaded_resources_contains_rds = resources.iter().any(|resource| {
+                matches!(
+                    resource.r#type,
+                    shuttle_common::resource::Type::Database(
+                        shuttle_common::database::Type::AwsRds(_)
+                    )
+                )
+            });
+
+            let resources: Vec<_> = resources
+                .into_iter()
+                .map(|resource| record_request::Resource {
+                    r#type: resource.r#type.to_string(),
+                    config: resource.config.to_string().into_bytes(),
+                    data: resource.data.to_string().into_bytes(),
                 })
                 .collect();
+
             resource_manager
                 .insert_resources(resources, &service_id, claim.clone())
                 .await
                 .expect("to add resource to persistence");
 
+            if !cached_resources_contains_rds && loaded_resources_contains_rds {
+                // TODO: this message should be generated by the resource type?
```

Comment:
I'm thinking of refactoring to an extension trait to extract prices and price_ids from resources.

Reply:
Although it's not ideal that the deployer knows the price/price_id of an item, that means we can't change it without the deployer being updated. I might rather hard-code a message for now, that the user should check the console to see the increased cost, and in the long run we can get it from a service we control, e.g. a call to auth to get billing details for a resource.

Reply:
Redirecting to the console sounds like a good idea for now

Reply:
Make deployer dumb again :tm: ;)